doc/user.xml

```diff
@@ -4607,6 +4620,35 @@ run</programlisting>
           playlist files.
         </para>
       </section>
+
+      <section id="youtube_dl_playlist">
+        <title><varname>youtube-dl</varname></title>
+
+        <para>
+          Downloads a song or playlist with <ulink
+          url="https://rg3.github.io/youtube-dl/">youtube-dl</ulink> from <ulink
+          url="https://rg3.github.io/youtube-dl/supportedsites.html">hundreds of
+          supported audio and video hosting sites</ulink>. <filename>youtube-dl
+          </filename> must be installed and available in the system <varname>
+          PATH</varname>, and the <link linkend="youtube_dl_playlist">youtube-dl
+          input plugin</link> must also be enabled. Add web URLs prefixed with
+          <filename>ytdl://</filename> to use it.
```

Comment:
The playlist and input plugin are distinct and use two different schemes, see that discussion point (partially because of add-then-load fallback behaviour, and partially because the input URI is considered an implementation detail and file add is inferior to the playlist load from a UX perspective even for single file additions).

If I can figure out what RemoteTagScanner is used for and how it works that may make adds with only the input plugin more user friendly if it does metadata loading in the background? I'm not really sure what it does.